### PR TITLE
Add Python 3.6 to Travis tests

### PR DESCRIPTION
**Pull request by wesleybowman**
I am unsure if this will work, but it would be nice if it did.

### CODE REVIEW

**Comment by wesleybowman** [comment]
@ocefpaf one last one. It would be nice to get python 3.6 tested as well. I added this, but I am not sure if it will work on Travis or not (not sure Travis can do 3.6 yet)

**Comment by wesleybowman** [comment]
Seems to pass!

File: .travis.yml [hunk 1]
```diff
@@ -13,6 +13,8 @@ matrix:
     env: TEST_TARGET=default
   - python: 3.5
     env: TEST_TARGET=coding_standards
+  - python: 3.6
+    env: TEST_TARGET=default
```
Comment (ocefpaf) [inline .travis.yml]: It should work, no reason not to.

Comment (wesleybowman) [reply .travis.yml]: Yeah, it passed. 

I figured I would merge this one, then we can make a `0.2.0` release

**Comment by ocefpaf** [comment]
Thanks @wesleybowman!

We need to update https://github.com/wesleybowman/UTide/blob/master/utide/__init__.py#L7

or, better yet, we can use `versioneer` and stop worrying about version numbers in the code. (`versionner` uses the git tag and inject it int he right places.)

Do you plan to update the PyPI package at https://pypi.python.org/pypi/UTide/0.1.0 ?

**Comment by wesleybowman** [comment]
I agree with using `versioneer`. 

I was doing all this right now on my lunch break, anything else will have to wait til after work.

**Comment by ocefpaf** [comment]
> I agree with using versioneer.

I can do the versionner PR, don't worry.

> I was doing all this right now on my lunch break, anything else will have to wait til after work.

Consider adding myself and/or @efiring to the PyPI team and GitHub repo we can probably help with the merges and releases.

**Comment by wesleybowman** [comment]
@ocefpaf what is your username on PyPI? `ocefpaf`?

**Comment by ocefpaf** [comment]
Yep `ocefpaf` everywhere :smile:

**Comment by wesleybowman** [comment]
ok, I have added you as a `maintainer` :+1:

**Comment by ocefpaf** [comment]
Thanks, once #51 is merged I'll try my new "powers" issue a `0.2.1` release.

**Comment by wesleybowman** [comment]
I also added you and @efiring as collaborators (when we started this, I was fairly new to git & github, time to straighten some things out)

**Pull request merged**